release-notes/release-notes54.md

```diff
@@ -43,23 +43,13 @@ For a list of:
 
 The following premium plugin updates were released alongside {{site.productname}} 5.4.
 
-### Accessibility Checker 2.x.x
+### Spell Checker Pro 2.1.0
 
-The {{site.productname}} 5.4 release includes an accompanying release of the **Accessibility Checker** premium plugin.
+The {{site.productname}} 5.4 release includes an accompanying release of the **Spell Checker Pro** premium plugin.
 
-**Accessibility Checker** 2.x.x 
-
-For information on the Accessibility Checker plugin, see: [Accessibility Checker plugin]({{site.baseurl}}/plugins/a11ychecker/).
-
-## Accompanying premium self-hosted server-side component changes
-
-The {{site.productname}} 5.4 release includes accompanying changes affecting the {{site.productname}} **self-hosted** services for the following plugins:
-
-- 
-
-
-### 
+**Spell Checker Pro** 2.1.0 provides better placement logic to prevent the Spell Checker dialog from covering the highlighted word. The change also allows users to interact with the document content without closing the Spell Checker dialog.
```

Comment:
> The change also allows users to interact with the document content without closing the Spell Checker dialog.

Is actually incorrect... as clicking in the editor will close the dialog (similar to all other inline dialogs).

Reply:
ok, deleting second sentence

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1571/commits/57fe3c68df59458c664fdee85ed5dff253d263d7